libs/s25main/desktops/dskCampaignSelection.cpp

```diff
@@ -88,6 +88,7 @@ dskCampaignSelection::dskCampaignSelection(CreateServerInfo csi)
     AddTextButton(ID_btBack, DrawPoint(380, 560), Extent(200, 22), TextureColor::Red1, _("Back"), NormalFont);
     AddTextButton(ID_Next, DrawPoint(590, 560), Extent(200, 22), TextureColor::Green2, _("Continue"), NormalFont);
 
+    GetCtrl<ctrlButton>(ID_Next)->SetEnabled(false);
```

Comment:
You can do this directly (not sure if this requires a line break then):
```suggestion
    AddTextButton(ID_Next, DrawPoint(590, 560), Extent(200, 22), TextureColor::Green2, _("Continue"), NormalFont)->SetEnabled(false);
```